applications/nftaa.md

```diff
@@ -0,0 +1,411 @@
+|    2d. | nftaa_increase/decrease | Functions needed to increase or decrease stake by NFTAA, this is needed for full compatibility with traditional methods of staking |
+
+### Milestone 3 — Web app + article
+
+- **Estimated Duration:** 1.5 month
+- **FTE:**  1.5
+- **Costs:** 7,000 USD
+
+| Number | Deliverable     | Specification                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+|-------:|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **0a.** | License                                      | MIT |
+| **0b.** | Documentation                                | Inline documentation of code, as well as startup configuration with all necessary commands, included in repository |
+| **0c.** | Testing and Testing Guide                    | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests. |
+| **0d.** | Docker                                       | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone. |
+|    0e. | Article         | We will publish an **article** that explains NFTAA (what was done/achieved as part of the grant). The article will be published through an online media service, e.g., medium.com, and advertised through social networks. Our ambition, as we are from academic environment, is to have also an academic paper about the idea of NFTAA in Polkadot published in a conference venue                                                                                    |
+|    3a. | Web app         | Develop a web application to interact with the nftaa_pallet, allowing users to create, manage, and transfer NFTAAs, provide a web application to interact with the nftaa_pallet: allowing users to create collections and single item; manage of metadata and attributes; operate with NFTAA ownership (transfers); play with staking, increase stake and unstake handled by NFTAA; read operations as get for an account, get assets of NFTAA, get the owner of NFTAA |
```

Comment:
We already have a ton of good staking interfaces and webapps. I don't think it makes sense to reinvent the wheel here. Instead I would suggest to integrate your necessary changes to a popular webapp like the Polkadot Staking Dashboard to make staking for NFTAAs work in a convenient way.

Reply:
We like this idea and we agree with you. This milestone can be understood as developing the integration to existing solutions, we definitely do not want to reinvent the wheel.

Reply:
great, would be nice if you could adapt the wording to that strategy.

Reply:
We will definitely do it :)

Reply:
Are you still working on this, @Roman-24?

Reply:
Changes have been made in  [this commit](https://github.com/w3f/Grants-Program/pull/2396/commits/1a8e20e1d464cf4d7e29e3da464d0673f3ca191d)

Reply:
nice.